packages/atlas/src/components/_video/VideoHero/VideoHero.tsx

```diff
@@ -77,7 +77,7 @@ export const VideoHero: React.FC<VideoHeroProps> = ({
       {headerNode && headerNode}
       <InfoContainer isCategory={isCategory}>
         <StyledLayoutGrid>
-          <GridItem colSpan={{ xxs: 12, xs: 10, sm: 6, md: 5, xl: 4, xxl: 3 }}>
+          <GridItem colSpan={{ xxs: 12, xs: 12, sm: 6, md: 6, xl: 6, xxl: 6 }}>
```

Comment:
If the number of columns stays the same, we need only to provide the smallest breakpoint, the rest is implied.
```suggestion
          <GridItem colSpan={{ xxs: 12, sm: 6 }}>
```